src/transformers/pipelines/image_to_text.py

```diff
@@ -92,11 +80,12 @@ def __call__(self, images: Union[str, List[str], "Image.Image", List["Image.Imag
 
                 The pipeline accepts either a single image or a batch of images.
 
-            max_new_tokens (`int`, *optional*):
-                The amount of maximum tokens to generate. By default it will use `generate` default.
+            prompt (`str`, *optional*):
+                Text to condition the generation on (only relevant for some models, like GIT).
 
-            generate_kwargs (`Dict`, *optional*):
-                Pass it to send all of these arguments directly to `generate` allowing full control of this function.
+            generate_kwargs:
+                Additional keyword arguments to pass along to the generate method of the model (see the generate method
+                corresponding to your framework [here](./model#generative-models)).
```

Comment:
`prompt` was not documented, and `max_new_tokens` is part of `generate_kwargs`

The docstring for `generate_kwargs` was copied from `TextGenerationPipeline`